cypress/e2e/integration.spec.js

```diff
@@ -128,7 +129,7 @@ describe('Nextcloud integration', function() {
 	it('Insert image', function() {
 		cy.get('@loleafletframe').within(() => {
 			cy.get('#Insert-tab-label').click()
-			cy.get('#insert-insert-graphic').click()
+			cy.get('#insert-insert-graphic-button').click()
 			cy.get('#w2ui-overlay-insert-graphic-menu .menu-text').eq(1).click()
```

Comment:
It's now failing because of this. Either replace it with `#insert-insert-graphic-entry-0` or `#insert-insert-graphic-entry-1`

![image](https://github.com/nextcloud/richdocuments/assets/65948705/00ef8279-f2f0-4754-9649-6c107053c5f4)